packages/react-instantsearch/examples/autocomplete/README.md

```diff
@@ -0,0 +1,8 @@
+This example shows how to build an autocomplete with react-instantsearch
+
+To start the example: 
+
+* npm install
+* npm start
+
+Read more about react-instantsearch [in our documentation](https://community.algolia.com/instantsearch.js/react/).
```

Comment:
> What I can explain on a guide is the idea behind the example: having a function listening to all changes and then construct the autocomplete state with it

OK that sounds of great help! Also I think you can mention that another specificity of this example is that it relies on a custom connector. This is very advanced, so let's be sure it's not too confusing.

About the robustness, it worries me a bit. Maybe it's not worth documenting at all unless we have a more solid solution? I don't know how much documentation are those examples either :) @vvo an idea on the subject?